src/test/ui/did_you_mean/recursion_limit.rs

```diff
@@ -13,6 +13,8 @@
 // fail the `Send` check by overflow when the recursion limit is set
 // very low.
 
+//~^^^^^^^^^^^^^^^ ERROR 1:1: 1:1: overflow evaluating the requirement
```

Comment:
I'm unclear on why the `1:1: 1:1` has popped up due to my changes, I must say.

Reply:
I agree, this is enough of a regression that I'd like to fix it before merging. Otherwise, the rest of the PR looks ok to me. I'd love to improve the wording (as mentioned in the other comments), but I don't really have a better proposal, and given that this PR isn't making them worse, I wouldn't block this PR on that.